lib/bakeAmbientOcclusion.js

```diff
@@ -332,7 +332,8 @@ function addVertexData(options) {
         },
         extras: {
             _pipeline: {
-                source: new Buffer(new Float32Array(allAOData).buffer)
+                source: new Buffer(new Float32Array(allAOData).buffer),
+                extension: 'application/octet-stream'
```

Comment:
This can be `.bin`. Should result in the correct uri type later in the pipeline.